lib/dataContract/stateTransition/DataContractCreateTransition.js

```diff
@@ -99,7 +137,7 @@ class DataContractCreateTransition extends AbstractStateTransitionIdentitySigned
  * @property {number} type
```

Comment:
`#fromJSON` should accept `JsonDataContractCreateTransition` so we need to define it (actually for all data structures)